src/main/java/com/coherentsolutions/training/automation/java/web/urnezaite/MyAccountPage.java

```diff
@@ -0,0 +1,49 @@
+package com.coherentsolutions.training.automation.java.web.urnezaite;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MyAccountPage extends BasePage {
+
+    private static final String TITLE = "My account - My Store";
+
+    @FindBy(css = "a.account>span")
+    private WebElement usersName;
+
+    @FindBy(xpath = "//a[@title = 'My wishlists']")
+    private WebElement wishListButton;
+
+    @FindBy(css = "a.home")
+    private WebElement homeButton;
+
+    public MyAccountPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public WishlistPage goToWishListPage() {
+        wishListButton.click();
+        return new WishlistPage(driver);
+    }
+
+    public HomePage goToHomePage() {
+        homeButton.click();
+        return new HomePage(driver);
+    }
+
+    public boolean isTitleCorrect() {
+        return TITLE.equals(driver.getTitle());
+    }
+
+    public boolean isCartButtonDisplayed() {
+        return cartButton.isDisplayed();
+    }
+
+    public boolean isWishListButtonDisplayed() {
+        return wishListButton.isDisplayed();
+    }
+
+    public boolean isUsersNameDisplayed() {
+        return usersName.isDisplayed();
+    }
```

Comment:
What's happen here if selenium doesn't find element? We will get NoSuchElement exception